### PR TITLE
fix typo

### PR DESCRIPTION
**Pull request by aneustroev**
##### SUMMARY
fix typo

##### ISSUE TYPE

- Docs Pull Request

##### COMPONENT NAME
community.general.proxmox_kvm

### CODE REVIEW

**Comment by felixfontein** [comment]
Thanks for fixing this! Since I'm not familiar with proxmox I'll keep this open until the beginning of next week and will merge then if nobody objects.

**Comment by aneustroev** [comment]
You can find api doc here https://pve.proxmox.com/pve-docs/api-viewer/index.html#/nodes/{node}/qemu/{vmid}/config , looks like misstype.

**Review by UnderGreen — APPROVED** [review]

**Comment by felixfontein** [comment]
@aneustroev thanks for fixing this!
@UnderGreen thanks for reviewing!

**Pull request merged**
